spring-integration-core/src/main/resources/org/springframework/integration/config/spring-integration-4.3.xsd

```diff
@@ -2691,6 +2691,34 @@
 		</xsd:complexType>
 	</xsd:element>
 
+	<xsd:element name="stream-transformer">
+		<xsd:annotation>
+			<xsd:documentation>
+				Defines a Consumer Endpoint for the
+				'org.springframework.integration.transformer.StreamTransformer'
+				that converts an 'InputStream' payload to a byte[] or String.
+				Providing a 'charset' signals that the conversion to String is
+				required.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:extension base="specialized-transformer-type">
```

Comment:
I based this on object-to-string-transformer - the only difference is the charset is not required here.

So it looks like that's wrong too - will take a look.